openfl/interface/envoy.py

```diff
@@ -31,8 +31,10 @@ def envoy(context):
 @envoy.command(name='start')
 @option('-n', '--shard-name', required=True,
         help='Current shard name')
-@option('-d', '--director-uri', required=True,
+@option('-da', '--director-addr', required=True,
```

Comment:
`director-addr` -> `director-host` or maybe only `host`, 'da' -> `h`

Reply:
`director-addr` -> `director-host`, but the director prefix remains to avoid confusion. Especially with the `-h` parameter, which is perceived as `--help`.